app/views/users/edit.html.slim

```diff
@@ -50,28 +50,29 @@
   = f.input :company_name, label: 'Company name'
   = f.input :company_info, label: 'Company info', hint: 'Where is the office, how much space can you provide, etc.'
   = f.input :bio, label: 'About you', hint: 'You can use basic <a href="http://en.wikipedia.org/wiki/Markdown" target="_blank">Markdown</a> here.'.html_safe
+
+  h3.page-header Technical Skills:
+  = f.input :tags, wrapper_html: { class: 'col-md-6' }, placeholder: 'Languages, techniques, etc involved', label: 'Technical_Expertise', hint: 'Separate tags by comma.'
+  = f.input :tags1, wrapper_html: { class: 'col-md-6' }, placeholder: 'Languages, techniques, etc involved', label: 'Technical_Interests', hint: 'Separate tags by comma.'
+
   h3.page-header I am interested in:
 
   = f.collection_check_boxes :interested_in, User::INTERESTS.to_a, :first, :last, item_wrapper_tag: 'div', item_wrapper_class: 'interested_in'
 
-  - if current_season.started?
```

Comment:
All the extra's are in your PR because of the problem with the syncing. As Carsten suggested, it is easier to close this PR, update your fork and make the changes again. Then this extra additions will not be in your PR.